CMakeLists.txt

```diff
@@ -89,9 +89,7 @@ else()
 endif()
 
 pkg_check_modules(LIBSOUP libsoup-2.4>=2.42)
-if (LIBSOUP_FOUND)
-	pkg_check_modules(JSON_GLIB REQUIRED json-glib-1.0)
-endif()
+pkg_check_modules(JSON_GLIB REQUIRED json-glib-1.0)
```

Comment:
This will only be required after #569 is merged.  I'd prefer if this change were part of that PR.

Reply:
ok i'll cancel this PR and send new one later